lib/engines/content_object_store/app/controllers/content_object_store/content_block/editions/steps_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ContentObjectStore::ContentBlock::Editions::StepsController < ContentObjectStore::BaseController
+  VALID_STEPS = {
+    review_links: "review_links",
+    schedule_publishing: "schedule_publishing",
+  }.freeze
+
+  def show
```

Comment:
I'm a bit confused about what the `update` method is for - it looks like `show` is the one that's used when updating?

Reply:
Ah I think I see, `show` and `update` hand off to each other - show shows the forms and `update` does any necessary actions?

Reply:
`show` is when we're showing the form, which then POSTs to the `update` method